cypress/component/integration/Dropdown.cy.tsx

```diff
@@ -0,0 +1,415 @@
+import '../../../src/global.css';
+import React from 'react';
+import { Dropdown, MassaLogo, MassaToken } from '../../../src/components';
+import { compareSnapshot } from '../../compareSnapshot';
+
+describe('Component | Integration | Dropdown', function () {
+  describe('Dropdown', () => {
+    describe('default dropdown', () => {
+      beforeEach(() => {
+        cy.mount(
+          <Dropdown
+            options={[
+              {
+                icon: (
+                  <MassaToken size={32} className="bg-c-default rounded-full" />
+                ),
+                item: 'account 1 name',
+                onClick: () => console.log('option 1'),
+              },
+              {
+                icon: <MassaLogo size={32} />,
+                item: 'account 2 name with a biiiiig content',
+              },
+              { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+              { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+              { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+              { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+            ]}
+          />,
+        );
+      });
```

Comment:
If we are mounting the SAME element I guess you can move this before each one level up then it will be valid for all tests down.